### DISCUSSION TITLE
v-alert warning icon display abnormal

### DISCUSSION THREAD

**Discussion by vikinghetty** [vuetifyjs/vuetify#13014] [General]
`<v-alert type="warning">hello world</v-alert>`

The type of info, error, success are fine, but type of warning is not a icon, display only a ugly "!"...it's different from the document.

how to fix it?

**Comment by kingyue737** [comment]
I also meet this issue in v2.5.8. The icon of warning alert is `mdi-exclamation` instead of `mid-alert`. The icon of error alert is `mdi-alert` instead of `mdi-alert-octagon`. It seems like a throw back to Vuetify 1.

![image](https://user-images.githubusercontent.com/40021217/139020913-d4ef2684-b4bd-4020-b9f9-dd861751f734.png)

It was identified as a bug #14327
